### PR TITLE
Web UI: Style and structure tweaks

### PR DESCRIPTION
**Pull request by scottymac**
Primary thrust of this PR is style tweaks to make the web ui just a tad more usable:
- Larger prompt box with a larger font size
- A bit more spacing between setting inputs
- Moving the form to the top of the page
- Allowing the image results to flow across entire viewport
- A little be better HTML organization (a fieldset each for `txt2img`, `img2img`, `gfpgan`, etc)

Besides styling this PR also changes some functionality:
- Clicking an image now opens the full size version in a new window
- Added the seed number below the image; clicking the seed number loads the seed into the current form

Hope this helps, but no biggie if you don't want to merge. In any case -- thanks for you hard work, @lstein! 

Video (Chrome): https://scottymac.s3.amazonaws.com/stable-diffusion/pr-images/sd-web-ui-update.mp4

<details><summary>Screenshots:</summary>

## Chrome
Screenshot (Chrome)

## Firefox
Screen Shot (Firefox)

## Safari
Screen Shot (Safari)

</details>

### CODE REVIEW

**Comment by bmaltais** [comment]
Nice update. It was always bugging me that the prompt field could not display the full prompt.

Now, my next pet peeve is that selecting an image should also allow you to make use of it as the input image for the img2img file selection ;-) Adding this would be awesome.

**Comment by heurihermilab** [comment]
Looks great!

**Comment by SMUsamaShah** [comment]
How does it look on mobile? I run server on pc and use from my phone. Being able to see prompt in full is great. Still want a few more things if you can.

1. Its nice having separate input elements for params but can you also allow entering full command in the textbox, just like we do in CLI. Reason for asking is so that i can easily copy paste prompts and args from lexica.art or logs or anywhere else.
2. a button to reset `choose file` box
3. (mobile) may be a checkbox to display images in full width, currently i have to zoom the page.
4. (mobile) if you are enable mobile view, please do not disable zooming as most websites do

**Comment by heurihermilab** [comment]
One observation — previously the displayed images had a rollover attribute containing the seed and prompt, now missing.

I found it handy but losing it wouldn't be a big problem (in my case).  Perhaps if there were a toggle to show the metadata...

Environment: MBP 14", M1 Pro, 16GB, latest OS, using miniforge with a base of Python 3.10.6.  Browser: Firefox 104.0.1.

**Comment by nls-smcmillin** [comment]
> One observation — previously the displayed images had a rollover attribute containing the seed and prompt, now missing.

@heurihermilab Good catch - I think that was the `title` attribute which I removed -- I'll add that back in, thanks!

**Comment by nls-smcmillin** [comment]
@heurihermilab Should be back now:

<details><summary>Screenshot:</summary>

Screen Shot 2022-09-04 at 6 51 43 PM

</details>

**Comment by scottymac** [comment]
(Oops posted last message with account -- going to delete that.)

@heurihermilab Should be back now:

<details><summary>Screenshot:</summary>

Screen Shot 2022-09-04 at 6 51 43 PM

</details>

**Comment by scottymac** [comment]
@bmaltais @SMUsamaShah All interesting ideas -- not sure how @lstein wants to handle suggestions for improvements (maybe in Issues?). I wanted this PR to be relatively low on functional changes. I could do some refactoring to make the page more mobile friendly in a future PR. And I definitely think a button to reset choose file input would be nice!

**Comment by heurihermilab** [comment]
One interface refinement — The "Generate" button works as a button but the cursor doesn't update to the "clicker" icon when rolling over it.

**Comment by tildebyte** [comment]
@scottymac; The improvements are definitely welcomed, but keep in mind that a complete re-write is underway, using React and a bunch of other stuff whose names I don't even know 😛.

tl;dr this PR will be temporary improvements to the existing web GUI until the "real" one is ready (preview https://peaceful-otter-7a427f.netlify.app). Of course you can coordinate with [psychedelicious](https://github.com/psychedelicious) on the new web GUI work if you want to - I'm sure he'd be happy to have more hands.

**Review by lstein — APPROVED** [review]
This looks good, and I'm merging. Would it be possible in a future revision to make the seed labels under the pictures a bit more prominent and add a visual indicator that clicking on the labels will restore the form fields to their earlier values? Otherwise I'm concerned that people won't figure that out. Maybe a thought bubble icon would do the trick?

**Comment by scottymac** [comment]
> make the seed labels under the pictures a bit more prominent and add a visual indicator that clicking on the labels will restore the form fields to their earlier values?

Yep, agreed -- there currently isn't an affordance to indicate the seed values are clickable. An icon of some sort would definitely help.
